soco/music_services/music_service.py

```diff
@@ -155,50 +196,138 @@ def call(self, method, args=None):
         try:
             result_elt = message.call()
         except SoapFault as exc:
+            if "Client.AuthTokenExpired" in exc.faultcode:
+                raise MusicServiceAuthException(
+                    "Authorization for {} expired or invalid: [{} / {} / {}]".format(
+                        self.music_service.service_name,
+                        exc.faultcode,
+                        exc.faultstring,
+                        exc.detail,
+                    )
+                ) from exc
```

Comment:
This fault code can occur when the token store does not contain auth info for a service. Should the case of missing credentials be reported to the user?

Reply:
Ah yes, that option should probably be explicitly mentioned in the error message.

Reply:
Added in 1a432b5